java/AxivityReader.java

```diff
@@ -312,14 +322,16 @@ private static void readCwaBuffer(ByteBuffer buf,
                     x = xRaw / 256.0;
                     y = yRaw / 256.0;
                     z = zRaw / 256.0;
-                    timeValue = getEpochMillis(blockTime);
```

Comment:
This is where the correction is called. `blockTime` which is a `LocalDateTime` object (no timezone) is zoned with DST correction (`zonedWithDSTCorrection()`), then converted to Unix time in milliseconds (milliseconds since 1 Jan 1970 UTC).